### PR TITLE
Добавлення Chapters 3

### PR DESCRIPTION
**Pull request by Dudiak-Nikolas**
Добавлення Глави 3, з готовим першим та другим завданням.

### CODE REVIEW

File: Chapters 3/Chapters 3.pyproj [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
```
Comment (auraz) [inline Chapters 3/Chapters 3.pyproj:1]: please move these types of files to gitignore.

File: Chapters 3/Exercise_1.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+def right_justify(s):
+    monty = ' ' * (70 - len(s)) + s
+    print(monty)
+right_justify('Monty')
```
Comment (auraz) [inline Chapters 3/Exercise_1.py]: please configure IDE to set newline at the end of file. Then red circle will be removed here.

File: Think_Python.sln [hunk 3]
```diff
@@ -3,10 +3,12 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 17
```
Comment (auraz) [inline Think_Python.sln]: Please move it to the gitignore

**Review by auraz — CHANGES_REQUESTED** [review]
Please address comments and add exercise 3.

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,14 @@
+root = true
```
Comment (auraz) [inline .editorconfig:1]: Please move to .gitignore

File: Chapters 3/Chapters 3.pyproj (hunk 1 above)
Comment (Dudiak-Nikolas) [reply Chapters 3/Chapters 3.pyproj:1]: I made changes to the commit :[b5e8de7]

File: Think_Python.sln (hunk 3 above)
Comment (Dudiak-Nikolas) [reply Think_Python.sln]: I made changes to the commit :[b5e8de7]

File: .editorconfig (hunk 4 above)
Comment (auraz) [reply .editorconfig:1]: Lets do it in the next PR

**Review by auraz — APPROVED** [review]

File: Chapters 3/Exercise_1.py (hunk 2 above)
Comment (Dudiak-Nikolas) [reply Chapters 3/Exercise_1.py]: added to the file .editorconfig code "insert_final_newline = true" to add a new line.

**Pull request merged**